packages/client/src/views/SysAdmin/Config/Application.tsx

```diff
@@ -96,81 +130,93 @@ function GeneralTabContent({
   const countryCurrencyName = lookup.currencies({
     code: offlineCountryConfiguration.config.CURRENCY.isoCode
   })
+  const items = [
+    {
+      label: intl.formatMessage(messages.applicationNameLabel),
+      value: offlineCountryConfiguration.config.APPLICATION_NAME,
+      action: {
+        id: GeneralActionId.APPLICATION_NAME,
+        label: intl.formatMessage(buttonMessages.change),
+        handler: () => {
+          callBack(GeneralActionId.APPLICATION_NAME)
+        }
+      }
+    },
+    {
+      label: intl.formatMessage(messages.govermentLogoLabel),
+      value: (
+        <img
+          src={offlineCountryConfiguration.config.COUNTRY_LOGO.file}
+          width={offlineCountryConfiguration.config.COUNTRY_LOGO_RENDER_HEIGHT}
+          height={offlineCountryConfiguration.config.COUNTRY_LOGO_RENDER_WIDTH}
+        />
+      ),
+      action: {
+        id: GeneralActionId.GOVT_LOGO,
+        label: intl.formatMessage(buttonMessages.change),
+        handler: () => {
+          callBack(GeneralActionId.GOVT_LOGO)
+        }
+      }
+    },
+    {
+      label: intl.formatMessage(messages.currencyLabel),
+      value: countryCurrencyName[0].name,
+      action: {
+        id: GeneralActionId.CURRENCY,
+        label: intl.formatMessage(buttonMessages.change),
+        handler: () => {
+          callBack(GeneralActionId.CURRENCY)
+        }
+      }
+    },
+    {
+      id: 'phoneNumberPattern_value_container',
+      label: intl.formatMessage(messages.phoneNumberLabel),
+      value: offlineCountryConfiguration.config.PHONE_NUMBER_PATTERN.toString(),
+      action: {
+        id: GeneralActionId.PHONE_NUMBER,
+        label: intl.formatMessage(buttonMessages.change),
+        handler: () => {
+          callBack(GeneralActionId.PHONE_NUMBER)
+        }
+      }
+    },
+    {
+      id: 'nidPattern_value_container',
+      label: intl.formatMessage(messages.nidPatternTitle),
+      value: offlineCountryConfiguration.config.NID_NUMBER_PATTERN.toString(),
+      action: {
+        id: GeneralActionId.NID_PATTERN,
+        label: intl.formatMessage(buttonMessages.change),
+        handler: () => {
+          callBack(GeneralActionId.NID_PATTERN)
+        }
+      }
+    }
+  ]
```

Comment:
We can probably clean up the code quite a bit if we map `GeneralActionId` directly like [FormTools](https://github.com/opencrvs/opencrvs-core/blob/c5e7b90bfd9ea39456ddd3bcd1ba8dcf373129b8/packages/client/src/components/formConfig/formTools/FormTools.tsx#L115-L134). But I guess for now as we are quite constrained in time maybe add a TODO?